_posts/2018-05-09-Stop-_-Frisk.md

```diff
@@ -26,21 +26,21 @@ The analysis also provides a number of interactive visualizations that can allow
+[Racial bias in the NYPD’s policing program has been studied extensively]( https://doi.org/10.1080/07418825.2012.712152), but incident-level data on the MPD's use of stop and frisk was only released earlier this year. The full dataset, [found here along with further documentation](https://mpdc.dc.gov/publication/stop-and-frisk-data-and-explanatory-notes) spans from 2010 through 2017 and contains demographic information about the person stopped as well as the time and location of the stop. 
 
-This data provides some opportunity to examine policing policy in the District, but ultimately falls well short of documenting MPD’s stop and frisk to the extent required by law.  In 2016, the DC City Council passed the Neighborhood Engagement Achieves Results Amendment Act of 2016, or NEAR Act. The legislation was designed to be a "comprehensive, public health-based approach to crime prevention and intervention", as described by its author [Kenyan R. McDuffie, Council-member for Ward 5](http://www.kenyanmcduffie.com/nearact/). Collecting richer data on felony crimes, stops and frisks, and use of force incidents was a key element of the law. Since 2016, [the MPD has failed to comply with this provision by omitting critical data on stop and frisk incidents, including what facts justified the stop, whether a frisk was initiated, and whether an arrest was made](https://www.wusa9.com/article/news/investigations/dc-police-not-following-law-requiring-stop-and-frisk-data-collection-2-years-after-it-was-passed/65-532319128).
+This dataset provides some opportunity to examine policing policy in the District, but ultimately falls well short of documenting MPD’s stop and frisk to the extent required by law.  In 2016, the DC City Council passed the Neighborhood Engagement Achieves Results Amendment Act of 2016, or the NEAR Act. The legislation was designed to be a "comprehensive, public health-based approach to crime prevention and intervention", as described by its author [Kenyan R. McDuffie, Council-member for Ward 5](http://www.kenyanmcduffie.com/nearact/). Collecting richer data on felony crimes, stops and frisks, and use of force incidents was a key element of the law. Since 2016, [the MPD has failed to comply with this provision by omitting critical data on stop and frisk incidents, including what facts justified the stop, whether a frisk was initiated, and whether an arrest was made](https://www.wusa9.com/article/news/investigations/dc-police-not-following-law-requiring-stop-and-frisk-data-collection-2-years-after-it-was-passed/65-532319128).
```

Comment:
GW: Rather than remove the sentence, I changed to mention the inconclusive findings from previous research, which is still valuable to have in my opinion.